components/PersonWidget/PersonWidget.tsx

```diff
@@ -19,6 +19,8 @@ interface Props {
 const PersonWidget = ({ person }: Props): React.ReactElement => {
   const dateOfBirth = prettyDate(person?.dateOfBirth ?? '');
 
+  const address = person?.addresses?.[0];
```

Comment:
probably this should be `displayAddress` or similar just for the sake of readability

Reply:
ok